packages/republik-crowdfundings/lib/scheduler/importPayments.ts

```diff
@@ -37,32 +34,45 @@ export async function importPayments(
   _args: undefined,
   context: Context,
 ): Promise<void> {
-  const transaction = await context.pgdb.transactionBegin()
-  let insertPaymentReport, unmatchedPaymentsAmount, matchingReport
-
   try {
-    await getAndInsertPostfinanceImports(transaction)
-    insertPaymentReport = await insertNewPayments(transaction)
-    matchingReport = await tryMatchingPayments(transaction, context)
-    await transaction.transactionCommit()
-    unmatchedPaymentsAmount = await getUnmatchedPaymentsAmount(context.pgdb)
+    await withTransaction(syncPaymentFiles, context.pgdb)
+    await withTransaction(_importPayments(context), context.pgdb)
+    await sendReminders()
   } catch (e) {
-    transaction.transactionRollback()
-    informFailed(
+    await informFailed(
       `importPayments(): postfinance sync failed with the following error: ${
         e.message || e
       } ${e.stack || ''}`,
     )
     return
   }
+}
+
+const _importPayments = (context: Context) => async (transaction: PgDb) => {
+  const insertPaymentReport = await insertNewPayments(transaction)
+  const matchingReport = await tryMatchingPayments(transaction, context)
+  const unmatchedPaymentsAmount = await getUnmatchedPaymentsAmount(transaction)
 
   await notifyAccountants({
```

Comment:
Now the transaction would be rolled back when `notifyAccountants` fails. Should be moved outside the transaction again.